elastic/test/test_elastic.py

```diff
@@ -523,7 +523,7 @@ def test_config_parser(self):
 
     def test_health_event(self):
         dummy_tags = ['foo:bar', 'elastique:recherche']
-        server_tags = ['cluster_name:elasticsearch']
+        cluster_tag = ["cluster_name:elasticsearch"]
```

Comment:
I think that for consistency, and since there is no interpolation we could keep the single quotes here?